iOSApp/Screens/Main/ViewController.swift

```diff
@@ -7,13 +7,52 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    @IBOutlet var tableView: UITableView!
+    
     override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        self.tableView.register(UINib(nibName: "CustomHeaderView", bundle: nil), forHeaderFooterViewReuseIdentifier: "CustomHeaderView")
+        self.tableView.register(UINib(nibName: "HorizontalRailsTableViewCell", bundle: nil), forCellReuseIdentifier: "HorizontalRailsTableViewCell")
```

Comment:
create new function `registerCells()` and ca.l if from `didSet` block from tableView outlet